### DISCUSSION TITLE
HFCs and PFCs, unspecified

### DISCUSSION THREAD

**Discussion by writetolalit** [USEPA/supply-chain-factors#56] [Q&A] [answered]
In version 1.2, data is reported for each NAICS code for "HFCs and PFCs, unspecified." For conversion to CO2e, what GWP should be used for this unspecified combo of GHGs?

**Comment by bl-young** [comment, marked as answer]
The units for that flow are incorrect and should be labeled as `kg CO2eq.`

See this [discussion post](https://github.com/USEPA/supply-chain-factors/discussions/41) and this [issue](https://github.com/USEPA/supply-chain-factors/issues/43).

The latest release ([v1.2.2](https://github.com/USEPA/supply-chain-factors/releases/tag/v1.2.2)) of the code produces outputs that are correct but a new published set of factors was not released.

**Reply by writetolalit** [reply]
Thank you for the clarification.
